modules/snooker/src/test/ts/browser/TableAdjustmentsTest.ts

```diff
@@ -102,167 +144,243 @@ UnitTest.test('TableAdjustmentsTest', () => {
-    testAdjustWidth(`ltr step (px) - resizeTable (1-1)`, 450, [ 100, 150, 100, 100 ], pixelTableWithColGroup(), 50, 1, resizeTable, true);
-    testAdjustWidth(`ltr step (px) - resizeTable (2-0)`, 450, [ 96, 96, 146, 96 ], pixelTable(), 50, 2, resizeTable, false);
-    testAdjustWidth(`ltr step (px) - resizeTable (2-1)`, 450, [ 100, 100, 150, 100 ], pixelTableWithColGroup(), 50, 2, resizeTable, true);
-    testAdjustWidth(`ltr large step (px) - resizeTable (0-0)`, 600, [ 296, 96, 96, 96 ], pixelTable(), 200, 0, resizeTable, false);
-    testAdjustWidth(`ltr large step (px) - resizeTable (0-1)`, 600, [ 300, 100, 100, 100 ], pixelTableWithColGroup(), 200, 0, resizeTable, true);
-    testAdjustWidth(`ltr large step (px) - resizeTable (1-0)`, 600, [ 96, 296, 96, 96 ], pixelTable(), 200, 1, resizeTable, false);
-    testAdjustWidth(`ltr large step (px) - resizeTable (1-1)`, 600, [ 100, 300, 100, 100 ], pixelTableWithColGroup(), 200, 1, resizeTable, true);
-    testAdjustWidth(`ltr large step (px) - resizeTable (2-0)`, 600, [ 96, 96, 296, 96 ], pixelTable(), 200, 2, resizeTable, false);
-    testAdjustWidth(`ltr large step (px) - resizeTable (2-1)`, 600, [ 100, 100, 300, 100 ], pixelTableWithColGroup(), 200, 2, resizeTable, true);
-    testAdjustWidth(`rtl step (px) - resizeTable (0-0)`, 350, [ 46, 96, 96, 96 ], pixelTable(), -50, 0, resizeTable, false);
-    testAdjustWidth(`rtl step (px) - resizeTable (0-1)`, 350, [ 50, 100, 100, 100 ], pixelTableWithColGroup(), -50, 0, resizeTable, true);
-    testAdjustWidth(`rtl step (px) - resizeTable (1-0)`, 350, [ 96, 46, 96, 96 ], pixelTable(), -50, 1, resizeTable, false);
-    testAdjustWidth(`rtl step (px) - resizeTable (1-1)`, 350, [ 100, 50, 100, 100 ], pixelTableWithColGroup(), -50, 1, resizeTable, true);
-    testAdjustWidth(`rtl step (px) - resizeTable (2-0)`, 350, [ 96, 96, 46, 96 ], pixelTable(), -50, 2, resizeTable, false);
-    testAdjustWidth(`rtl step (px) - resizeTable (2-1)`, 350, [ 100, 100, 50, 100 ], pixelTableWithColGroup(), -50, 2, resizeTable, true);
-    testAdjustWidth(`rtl large step (px) - resizeTable (0-0)`, 314, [ 10, 96, 96, 96 ], pixelTable(), -200, 0, resizeTable, false);
-    testAdjustWidth(`rtl large step (px) - resizeTable (0-1)`, 310, [ 10, 100, 100, 100 ], pixelTableWithColGroup(), -200, 0, resizeTable, true);
-    testAdjustWidth(`rtl large step (px) - resizeTable (1-0)`, 314, [ 96, 10, 96, 96 ], pixelTable(), -200, 1, resizeTable, false);
-    testAdjustWidth(`rtl large step (px) - resizeTable (1-1)`, 310, [ 100, 10, 100, 100 ], pixelTableWithColGroup(), -200, 1, resizeTable, true);
-    testAdjustWidth(`rtl large step (px) - resizeTable (2-0)`, 314, [ 96, 96, 10, 96 ], pixelTable(), -200, 2, resizeTable, false);
-    testAdjustWidth(`rtl large step (px) - resizeTable (2-1)`, 310, [ 100, 100, 10, 100 ], pixelTableWithColGroup(), -200, 2, resizeTable, true);
-  };
+  context('preserve table column resizing', () => {
```

Comment:
As it's hard to see what was added here, look for anything that has `overflow` in the title.